src/kixi/gwyn/gwyn.clj

```diff
@@ -39,19 +40,53 @@
    (-> fire-station-lookup-table
        (wds/select-from-ds {:station fire-station})
        (wds/subset-ds :cols [:radius :lat :long]))})
+
+(def google-places-api "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=")
+
+(defn get-env-var [key]
+  (get (System/getenv) "places_api_key"))
```

Comment:
Also, `(System/getenv key)` avoids returning the entire environment (only returns the key you want). Also, env vars are idiomatically uppercase i.e. `PLACES_API_KEY`.